raiden/transfer/events.py

```diff
@@ -205,24 +205,47 @@ class EventTransferSentSuccess(Event):
         sucessful but there is no knowledge about the global transfer.
     """
 
-    def __init__(self, identifier, amount, target):
+    def __init__(
+            self,
+            payment_network_identifier,
+            token_network_identifier,
+            channel_identifier,
+            identifier,
+            amount,
+            target,
+    ):
+        self.payment_network_identifier = payment_network_identifier
+        self.token_network_identifier = token_network_identifier
+        self.channel_identifier = channel_identifier
         self.identifier = identifier
         self.amount = amount
         self.target = target
 
     def __repr__(self):
-        return '<EventTransferSentSuccess identifier:{} amount:{} target:{}>'.format(
+        return (
+            '<'
+            'EventTransferSentSuccess identifier:{} amount:{} '
+            'target:{} payment_network_identifier:{} channel_identifier:{} '
+            'token_network_identifier:{}'
+            '>'
+        ).format(
             self.identifier,
             self.amount,
             pex(self.target),
+            self.payment_network_identifier,
```

Comment:
IMO this would be better if it was consistent with the other `__repr__` where the representation follow the order of the constructor arguments.